v_4_9_0/files/config/kubelet-master.yaml.tmpl

```diff
@@ -20,7 +20,6 @@ kubeReserved:
   memory: 1280Mi
   ephemeral-storage: 1024Mi
 kubeReservedCgroup: /kubereserved.slice
-kubeletCgroups: /kubereserved.slice
```

Comment:
That is something we should fix instead of removing

Reply:
This is what I'm trying to test now. Kubelet still runs in the kubereserved slice without this as it's in the systemd unit. How can you check if this affects the resource calculation in Kubernetes?

Reply:
Why can’t we check what is wrong with logs instead of calculating allocatable?
There is some bug which we first need to prove, then decide on a way to proceed.

Reply:
I’m happy to help if you give me a cluster tomorrow

Reply:
I can reliably reproduce the bug by toggling this `kubeletCgroups: /kubereserved.slice` in the kubelet config. `journalctl -u k8s-kubelet` only shows the first two seconds of logs and then stops. `journalctl -o verbose` shows the missing kubelet log lines without the SYSTEMD_UNIT annotation. I'm not sure if it's a systemd bug or some improper configuration. Help tomorrow would be appreciated.

Reply:
I've looked into this and didn't find what could be wrong. But even removing `kubeletCgroups` didn't fix the issue for me. It freezes every time

Reply:
What are our options here?

We need to pick this up again soonish for 1.16 but not having logs in journalctl is unacceptable. Is there somewhere upstream where we could raise this?

Reply:
I've run a quick search through upstream issues and I could not find anything. Perhaps we need to dig more on upstream and open it there.

Reply:
I am leaning towards it being a systemd bug. I was thinking of looking in the source code to see how kubelet changes its own cgroup and see if I can reproduce without kubelet. I get the impression that these advanced cgroup options aren't used very often.